prettier/Dockerfile

```diff
@@ -1,7 +1,7 @@
 FROM node:6.9.1-slim
 MAINTAINER Pat Brisbin <pbrisbin@gmail.com>
 ENV LANG en_US.UTF-8
-ENV PRETTIER_VERSION 1.19.1
+ENV PRETTIER_VERSION 2.0.2
```

Comment:
You'll want to update the `info.yaml` as well. We tag the images to convey the underlying tools version.

Reply:
I updated it.